### ISSUE TITLE
Create a release for f-droid

### ISSUE THREAD

**Issue by hammerandtongs** [abarisain/dmix#869]
https://gitlab.com/fdroid/fdroiddata/-/issues/2058#note_354969201

The wifi android 9 fix isn't in f-droid yet.

**Comment by abarisain** [comment]
Does it even compile? Last time i checked toils were so out of date just running the project crashed on a phone

I'm sorry but mpdroid is a dead project. I'll try compiling master and run it using android studio 4.0, but if it fails I'm not gonna tag a broken release. Unfortunately this patch has been developed against master which is way behind dev in terms of features
I'll keep you posted.

Fdroid maintainers have been quite rude to me in the past, so I don't want anything to do with them. Seeing how your issue there got closed doesn't make it sound like they changed.

**Comment by NHellFire** [comment]
@abarisain I updated the project files last year for the latest SDK (and updated/replaced obsolete libraries). I think I still need to update the library list in about. I also need to split the commits up and recheck it, but due to life/health issues I haven't had a chance to. I'm hoping I can get to that soon and make a PR. The current release on Play Store at least still works on Android 10.

**Comment by abarisain** [comment]
Yeah, the current release works in compatibility mode, I guess.  

I still don't maintain this, so please don't spend a lot of precious time on the PR as I don't even know when I'll merge it. 

That said, if you got it to work on modern android when targeting a recent SDK, that's awesome. Because the master branch doesn't even compile on a modern Android Studio on my computer ^^'
